### PR TITLE
Add personal walls

### PR DESCRIPTION
**Pull request by j-dav**

### CODE REVIEW

**Comment by j-dav** [comment]
[make post on wall [MEDIUM]](https://trello.com/c/VkocbMdB/63-make-post-on-wall-medium)

**Comment by j-dav** [comment]
[wall stream - set up new routes [SMALL]](https://trello.com/c/bCFSrqbJ/60-wall-stream-set-up-new-routes-small)

**Comment by j-dav** [comment]
[wall stream - make posts#index filter by recipient_id [MEDIUM]](https://trello.com/c/Jsz0AU5Z/62-wall-stream-make-postsindex-filter-by-recipientid-medium)

**Review by JRRS1982 — APPROVED** [review]
Looking great! Amazing effort chaps!

File: spec/features/posts_are_ordered_spec.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@
 
 RSpec.feature "Posts are ordered", type: :feature do
   scenario "Posts are ordered by newest first" do
-    sign_up
-    create_post message: "First post"
-    create_post message: "Second post"
+    sign_up username: "test_user"
+    create_post on_wall_of: "test_user", message: "First post"
```
Comment (arthurashman) [inline spec/features/posts_are_ordered_spec.rb:8]: This code is particularly readable, good job

**Comment by arthurashman** [comment]
Looks great, happy to merge
Jeremy Agrees

**Pull request merged**